tests/cpp-tests/Classes/ExtensionsTest/CocoStudioActionTimelineTest/ActionTimelineTestScene.cpp

```diff
@@ -66,11 +66,11 @@ void TestActionTimeline::onEnter()
 {
     ActionTimelineBaseTest::onEnter();
 
-    Node* node = CSLoader::createNode("ActionTimeline/DemoPlayer.csb");
-    ActionTimeline* action = CSLoader::createTimeline("ActionTimeline/DemoPlayer.csb");
+    Data data = FileUtils::getInstance()->getDataFromFile("ActionTimeline/DemoPlayer.csb");
+    Node* node = CSLoader::createNodeWithDataBuffer(data);
+    ActionTimeline* action = CSLoader::createTimelineWithDataBuffer(data, "ActionTimeline/DemoPlayer.csb");
```

Comment:
OK, I see. Keep the `key`  string arg.
But `createBy`, `createWith` is not recommended, replace with `create` should be better, as well as `createTimelineWithDataBuffer` may be get timeline from cache, so `WithDataBuffer` is not a good idea.